static/503.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Code City</title>
+  <style>
+    body {
+      font-family: "Roboto Mono", monospace;
+      text-align: center;
+    }
+    h1 {
+      font-size: 40pt;
+      font-weight: 100;
+    }
+    h1>svg {
+      vertical-align: text-bottom;
+    }
+    #tagline {
+      font-style: italic;
+      margin: 2em;
+    }
+    iframe {
+      height: 50px;
+      width: 100px;
+      border: none;
+      display: block;
+      margin: 0 auto;
+    }
+  </style>
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
+  <link href="/static/favicon.ico" rel="shortcut icon">
+</head>
+<body>
+  <h1>
+    <svg xmlns="http://www.w3.org/2000/svg" width="95px" height="100px">
```

Comment:
If you approve/merge PR #116, then this \<svg\> can be replaced with:
`    <img src="/static/logo-error.svg" width="95px" height="100px" alt="">`
Also edit `h1>svg` in the CSS above to be `h1>img`

Reply:
Done.